certbot/tests/auth_handler_test.py

```diff
@@ -191,15 +192,73 @@ def test_name3_http_01_3_common_acme_2(self):
 
     def test_debug_challenges(self):
```

Comment:
Thinking about the value of these 3 tests, maybe they could be simplified to:

1. The original test, which checks that `The following` is not printed without `-v`
2. A test for `-v` with two authzrs (one HTTP and DNS challenge in each),  asserting that the all the expected URL, FQDNs and keyAuthzs are all present.